### ISSUE TITLE
JavaScript Error after Sleek closing

### ISSUE THREAD

**Issue by DmDrMos** [ransome1/sleek#402]
![image](https://user-images.githubusercontent.com/103596016/182045747-139bfe93-5030-4954-a7c5-bdc76cb8da7c.png)

**Describe the bug**
The error message occurs every time the application is closed/

**Do you see any error entries in sleeks developer tools?**
I have a message "Uncaught (in promise) Error: Error invoking remote method 'userData': No handler registered for 'userData'" in the developer console

**Desktop (please complete the following information):**
 - OS: [Windows 11]
 - Version of sleek [1.2.1]
 - Source: Github,

**Comment by ransome1** [comment]
Thanks for reporting this. Does this also happen with v1.2.0?

Unfortunately I cannot look into it at the moment as I'm afk until October. But I'll keep you posted as soon as there is progress on this.

**Comment by berkes** [comment]
![Schermafdruk van 2022-08-04 09-17-43](https://user-images.githubusercontent.com/77059/182787445-638c0496-dd04-4424-ac63-d8f7292277dd.png)

Same error on Ubuntu snap.

```
❯ snap info sleek
name:    sleek
summary: sleek is an open-source (FOSS) todo manager based on the todo.txt
  syntax. It's available for Linux
publisher: Robin Ahle (ransome)
store-url: https://snapcraft.io/sleek
contact:   https://github.com/ransome1/sleek
license:   MIT
description: |
  sleek is an open-source (FOSS) todo manager based on the todo.txt syntax.
  Stripped down to only the most necessary features, and with a clean and
  simple interface, sleek aims to help you focus on getting things done.
  
  All classic todo.txt attributes are supported and enhanced by additional
  features. Creating todos is straightforward, and tag-based filtering in
  tandem with highly customisable grouping and smart full-text search allow
  for rapid information retrieval. Completed todos can be hidden or archived
  into separate done.txt files. Easy integration with other todo.txt apps is
  facilitated by continuously scanning todo.txt files for changes.
  
  sleek is available for Windows, MacOS and Linux, and in several languages.
  For a detailed list of features, see below.
  
  ### Features
  * Uses existing todo.txt files or creates new ones
  * Add and search for todos by
    - priorities
    - contexts
    - projects
    - due dates
    - creation dates
    - recurrences (repeating todos)
    - threshold dates
  * Sort and group todos by priority, due and creation date, context and
  project or as they occur in the text file
  * Filter todos by context, project and priority
  * Find todos using full-text search compatible with todo.txt syntax
  * Inline autocomplete available
  * Dates and priorities can be selected by built-in picker elements
  * Navigable via keyboard shortcuts
  * Tabindex available
  * Options for due date reminders and notification badges
  * Easily toggle between dark and light mode
  * Compact view and zoom available
  * Completed todos can be shown, hidden and archived
  * Multi line todos can be created
  * Filters are sorted alphanummerically and can be renamed or deleted
  * Hyperlinks detected automatically
  * File watcher scans todo.txt files for changes
  * Simultaneously manage multiple todo.txt files
  * Language options
    - English
    - German
    - Italian
    - Spanish
    - French
    - Simplified Chinese
    - Brazilian Portugese
    - Japanese
    - Turkish
    - Hungarian
    - Czech
    - Polish
commands:
  - sleek
snap-id:      valFazwDN1HYt1gC3SgmmyvNVoCYzjxe
tracking:     latest/stable
refresh-date: today at 09:15 CEST
channels:
  latest/stable:    1.2.0 2022-06-13 (1041) 113MB -
  latest/candidate: ↑                             
  latest/beta:      ↑                             
  latest/edge:      1.2.1 2022-06-15 (1048) 115MB -
installed:          1.2.0            (1041) 113MB -
```

So, yes. 1.2.0 has this.

**Comment by ransome1** [comment]
@berkes @DmDrMos At the moment I can only work on a M1 Mac (VirtualBox doesn't run yet on these machines, so I can't access my VMs). 

Anyways, can I ask you guys to try out 1.2.2 and tell me if this still happens? It should be available on Snap or downloaded here: https://github.com/ransome1/sleek/releases/tag/v1.2.3-rc.1

**Comment by DmDrMos** [comment]
version": "1.2.4-rc.1
After closing main window. OS version Windows 11.
![image](https://user-images.githubusercontent.com/103596016/193609825-30b59f09-f3d8-4222-9e2a-2edf95a85384.png)

**Comment by ransome1** [comment]
@DmDrMos 
> version": "1.2.4-rc.1 After closing main window. OS version Windows 11. ![image](https://user-images.githubusercontent.com/103596016/193609825-30b59f09-f3d8-4222-9e2a-2edf95a85384.png)

This one should be fixed since 1.2.4-rc.2. Please try to use https://github.com/ransome1/sleek/releases/tag/v1.2.5 and let me know if it still happens.

@berkes
> ![Schermafdruk van 2022-08-04 09-17-43](https://user-images.githubusercontent.com/77059/182787445-638c0496-dd04-4424-ac63-d8f7292277dd.png)
> 
> Same error on Ubuntu snap.
> 
> So, yes. 1.2.0 has this.

This is actually a different error. Can you tell me when exactly this happens? Also, can you please download 1.2.5? It should be available on Snap.
